ow_lander/README.md

```diff
@@ -1,84 +1,149 @@
-URDF and related code. Please prefer this header over writing similar code
-yourself to keep all our code consistent. If the lander is modified to change
-the number of joints or any joint names, this header must be updated as well.
-
-Run it
-------
-To run along with Gazebo:
-
-`$ roslaunch ow europa_terminator_workspace.launch`
+## Usage
 
-This will start Gazebo, as well as the planning and feeding nodes. Now to run
-both these nodes, you need to call the appropriate services (note that PLEXIL
-will be calling these services in code).
+First run the simulation using any of the available OceanWATERS launch configurations:
 
-### Unstow trajectory planning
-Call the Unstow service.
-
-### Guarded move trajectory planning
-Call the GuardedMove service. This creates two trajectories in .ros.
+```bash
+roslaunch ow atacma_y1a.launch  # other options:
+                                # * europa_terminator.launch
+                                # * europa_terminator_workspace.launch
+```
 
-### Grind trajectory planning
-Call the Grind service.
+This will start the simulation, as well as loads available services to to control
+the arm, you can interact with the exposed services using any of the two methds
+described below:
 
-### Circular trenching trajectory planning
-Call the DigCircular service.
+### Calling Arm Operations using RQT Service Caller
+When OceanWATERS sim is launched by default it starts an instance of **rqt_gui**
+that has the _Service Caller_ plugin enabled. You can use the service caller
+interface to invoke the different arm services that are exposed by the simulation.
+Find the **rqt_gui** window then select the _Service Caller_ tab. Using the drop
+down menu, select the service that you want to invoke (for example`/arm/dig_circular`
+or `/arm/guarded_move`), configure the service arguments, as shown below:
+<p align="center"><img src="./misc/service_caller.png" width="40%" ></p>
 
-### Linear trenching trajectory planning
-Call the DigLinear service.
+Finally, press the `Call` button to invoke the service as configured.
 
-### Deliver sample trajectory planning
-Call the DeliverSample service.
+### Calling Arm Operations using ROS Command Line Interface
+Alternatively to using the **RQT Service Caller** gui interface, you may interact
+with the services using the `rosservice` command line tool. While the simulation
+is running, start a new terminal and source OceanWATERS workspace. Then, invoke
+the services using their respective commands as listed below:
 
-### Stow trajectory planning
-Call the Stow service.
+> Note: The provide examples show running these service using default values. 
 
-### Trajectory feeding
-Call the PublishTrajectory service.
+* Stow
+```bash
+rosservice call /arm/stow "{}"
+```
+* Unstow
+```bash
+rosservice call /arm/unstow "{}"
+```
+* Deliver Sample
+```bash
+rosservice call /arm/deliver_sample \
+  "{use_defaults: true,
```

Comment:
I don't understand why, but every 'use_defaults: true' line appears red in my Markdown viewer.

Reply:
I don't have this issue, change your markdown viewer :wink:

Reply:
I don't have this issue using the Markdown Viewer extension in Chrome.